weld/sir.rs

```diff
@@ -355,11 +374,12 @@ impl fmt::Display for ParallelForIter {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         if self.start.is_some() {
             write!(f,
-                   "iter({}, {}, {}, {})",
+                   "iter({}, {}, {}, {}, {})",
                    self.data,
                    self.start.clone().unwrap(),
                    self.end.clone().unwrap(),
-                   self.stride.clone().unwrap())
+                   self.stride.clone().unwrap(),
+                   self.kind)
         } else {
```

Comment:
Nit: add another case for when self.kind is set so that we see fringe vs vectorized in the SIR programs